Makefile.common

```diff
@@ -60,6 +60,7 @@ BASE_FLAGS = \
   -minimal \
   -add-include '"kremlin/internal/types.h"' \
   -add-include '"kremlin/lowstar_endianness.h"' \
+  -add-include '"kremlin/fstar_int.h"' \
```

Comment:
can you use the new syntax (see kremlin/Changes.md) to insert this only for the QTesla file?